src/pages/about.jsx

```diff
@@ -1,5 +1,82 @@
+import React from 'react'
+import InfoItem from '../components/infoItem'
+import EmailSignUp from '../components/emailSignUp'
+import feature2 from '../assets/images/features1.png'
+import feature3 from '../assets/images/features3.png'
+
 function AboutPage() {
-  return <div>This is About Page</div>
+  return (
+    <div>
+      {/* Page header*/}
+      <div className="">
+        <p className="laptop:text-9 laptop:pt-16 laptop:pb-20 laptop:px-[450px]  text-h2 py-8 px-9 ">
+          A brand built on the love of craftmanship, quality and outstanding
+          customer service
+        </p>
+      </div>
+      {/* Features sections */}
+      <div className="grid grid-cols-1 laptop:grid-cols-2 mx-auto">
+        {/* Feature 1 */}
+        <div>
+          <div className="px-6 mt-12 sm:mb-9 laptop:px-28  laptop:pt-20 ">
+            <div className="text-h4 laptop:text-6">
+              From a studio in London to a global brand with over 400 outlets
+            </div>
+            <p className="font-satoshi text-3.5 mt-3 laptop:text-4">
```

Comment:
replace "font-satoshi text-3.5 laptop:text-4" with "text-body-sm laptop:text-body-md". I have declared 3 more text styles that applies font satoshi in index.css

Reply:
Done